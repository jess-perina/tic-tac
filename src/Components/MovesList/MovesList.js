import React from 'react'

function MovesList(props) {
  return (
    <ul className="collection">{props.moves}</ul>
  )
}

export default MovesList
