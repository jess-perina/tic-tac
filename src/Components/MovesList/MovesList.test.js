import React from 'react'
import { shallow } from 'enzyme'
import MovesList from './MovesList'

describe('MovesList Component', () => {
  it('renders as espected', () => {
    expect(shallow(<MovesList />)).toMatchSnapshot()
  })
})