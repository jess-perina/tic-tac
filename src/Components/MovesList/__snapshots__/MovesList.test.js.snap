// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovesList Component renders as espected 1`] = `ShallowWrapper {}`;
